util/createErrorResponse.ts

```diff
@@ -13,8 +13,11 @@ interface ErrorResponse {
 const createErrorResponse = (error: Err, context: Context): ErrorResponse => {
   context.log(error);
 
-  if (error?.message === 'DataStorage not found') {
-    context.log('no datastorage found, returning empty array');
+  if (
+    error?.message === 'User has no datastorage' ||
+    error?.message === 'Could not get datastorage'
```

Comment:
With code like this, there is a high risk of this breaking at some point but lets just keep it as is for now.